troposphere/static/js/components/modals/instance/launch/components/ProjectGroup.react.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import SelectMenu from 'components/common/ui/SelectMenu.react';
+
+export default React.createClass({
+
+    render: function() {
+        if (this.props.projectList) {
```

Comment:
Yes, that was disabled while developing. This is incomplete, it is the beginnings of the new feature:  To handle no projects by providing a prompt to create a new project if none exist. 

We just merged the state I was at because Connor had fixed a bug introduced by the last refactor.